### PR TITLE
Added simple script for running all tests in NTVS

### PR DESCRIPTION
**Pull request by mjbvz**
Adds a simple powershell script that can be used to run all tests that are part of NTVS.

I'm also working on documenting how contributors can run our tests on the wiki: https://github.com/Microsoft/nodejstools/wiki/Running-Our-Tests

### CODE REVIEW

**Comment by kant2002** [comment]
Great thanks for your effort. It really helps!

File: runtests.ps1 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Run the basic sets of tests using powershell
+$vstest = "C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\CommonExtensions\Microsoft\TestWindow\vstest.console.exe"
```
Comment (mousetraps) [inline runtests.ps1]: This script will not work for multiple versions of Visual Studio.

File: runtests.ps1 [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Run the basic sets of tests using powershell
+$vstest = "C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\CommonExtensions\Microsoft\TestWindow\vstest.console.exe"
+
+$tests = @("NpmTests", "NodeTests", "ProfilerTests", "JSAnalysisTests", "MockVsTests", "Nodejs.Tests.UI")
```
Comment (mousetraps) [inline runtests.ps1]: We should also run SharedProjectTests and add a comment that AzurePublishing.Tests.UI are being skipped.

File: runtests.ps1 [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Run the basic sets of tests using powershell
```
Comment (mousetraps) [inline runtests.ps1]: This file shouldn't be top-level - can you move this into a directory within Tools?

**Comment by mjbvz** [comment]
Ok, updated based on your comments.

File: Tools/runtests.ps1 [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+$tests = @("JSAnalysisTests", "Nodejs.Tests.UI", "NodeTests","NpmTests", "ProfilerTests", "SharedProjectTests")
+
+# Try to get the path to the vstest.console test runner
+$vstest =  ''
+foreach($vsversion in $vsversions) {
+	$path = "C:\Program Files (x86)\Microsoft Visual Studio $vsversion\Common7\IDE\CommonExtensions\Microsoft\TestWindow\vstest.console.exe"
+	if (Test-Path $path) {
+		$vstest = $path
+	}
+}
```
Comment (mousetraps) [inline Tools/runtests.ps1]: what if there are multiple VS versions installed?

File: Tools/runtests.ps1 [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+if ($vstest -eq '') {
+	echo "Could not find valid vstest.console"
+	return
+}
+
+$testFiles = $tests | % { ".\BuildOutput\Release14.0\Tests\$_.dll" }
```
Comment (mousetraps) [inline Tools/runtests.ps1]: this is still targeting 14.0

File: Tools/runtests.ps1 (hunk 4 above)
Comment (mjbvz) [inline Tools/runtests.ps1]: Thrice the testing!

**Comment by mousetraps** [comment]
:+1: looks good - we can continue to iterate on the script as we need more features (accepting different args, etc.)

**Pull request merged**